src/Home/Home.js

```diff
@@ -38,6 +40,7 @@ function mapStateToProps(state){
     return{
         isModal_zipcode_shown: state.AppReducer.get('isModal_zipcode_shown'),
         isModal_post_shown: state.AppReducer.get('isModal_post_shown'),
+        isModal_signIn_shown: state.AppReducer.get('isModal_signIn_shown'),
```

Comment:
Can you combine these all here to just be, `is_modal_shown`? Do you need to know which one it is?

Reply:
I figured that the main purpose for all these state props are to determine whether the page mask should be shown. So created a new state prop in App reducer called 'IsPageMaskShown", which is updated whenever a modal (any modals) are shown. Any modals shown would be the page mask is shown. So this should answer why I need that props in the app reducer (your later comment)